Permanent/ViewModels/FilesViewModel.swift

```diff
@@ -221,6 +223,55 @@ extension FilesViewModel {
 
     }
     
+    func getRecord(file: FileViewModel, then handler: @escaping (RecordVO?) -> Void) {
```

Comment:
We don't have to reimplement this function. We should be able to use the DownloadManagerGCD.download function that takes in a FileDownloadInfo. 

The above mentioned function already has this logic implemented: it gets the record, then in starts a download based on that record. 

Doing this should also simplify the ViewController logic as well